.github/workflows/tests.yml

```diff
@@ -47,11 +47,9 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-python@v2
       - run: pip install tox
-      - name: Patch Buildkite-specific test script
```

Comment:
Wasn't sure what @callahad had in mind here (see line 41 above) for simplifying the check-newsfragment script.

Aside: I can't seem to leave a comment on line 41 because(?) it's not part of the diff hunk?

Reply:
> Aside: I can't seem to leave a comment on line 41 because(?) it's not part of the diff hunk?

welcome to github PR review!

Reply:
... of course you can make it part of the diff hunk by removing the comment about Buildkite :). I think you've done is all that was needed here.

I'm not sure how it relates to the issues about checkout though. I guess they are meant to be a separate comment.

Reply:
Please can you remove the comment at line 41?